runtime/default.policy

```diff
@@ -1,82 +1,133 @@
+      open.filename =~ "/etc/pam.d/*" && open.flags & (O_RDWR | O_WRONLY | O_CREAT) > 0
+
+  - id: cron_at_job_creation
     description: Unauthorized scheduling client
     expression: >-
       open.filename =~ "/var/spool/cron/*" && open.flags & (O_CREAT | O_RDWR | O_WRONLY) > 0 &&
-      process.name not in ["at", "crontab"]
-  - id: kernel_modification
-    description: Unauthorized kernel modification
-    expression: >-
-      open.filename =~ "/boot/*" && open.flags & (O_CREAT | O_RDWR | O_WRONLY) > 0
+      process.filename not in ["/usr/bin/at", "/usr/bin/crontab"]
+
   - id: systemd_modification
     description: Unauthorized modification of a service
     expression: >-
-      (open.filename =~ "/lib/systemd/system/*" || open.filename =~ "/usr/lib/systemd/system/*") && open.flags & (O_CREAT | O_RDWR | O_WRONLY) > 0
+      (open.filename =~ "/lib/systemd/system/*" || open.filename =~ "/usr/lib/systemd/system/*") &&
+      open.flags & (O_CREAT | O_RDWR | O_WRONLY) > 0
+
   - id: authentication_logs_accessed
     description: unauthorized file accessing access logs
     expression: >-
       open.filename in ["/run/utmp", "/var/run/utmp", "/var/log/wtmp"] &&
-      process.name not in ["login", "sshd", "last", "who", "w", "vminfo", "sudo"]
-  - id: root_ssh_key
-    description: attempts to create or modify root's SSH key
-    expression: >-
-      open.filename == "/root/.ssh/authorized_keys" && open.flags & (O_CREAT | O_RDWR | O_WRONLY) > 0
+      process.filename not in ["/usr/bin/login", "/usr/sbin/sshd", "/usr/bin/last", "/usr/bin/who", "/usr/bin/w",
+      "/usr/bin/sudo", "/usr/sbin/cron"]
+
+  - id: ssh_authorized_keys
+    description: attempts to create or modify an authorized_keys file
+    expression: >-
+      (open.filename =~ "/root/.ssh/" || open.filename =~ "/home/*") && open.basename == "authorized_keys"
+       && open.flags & (O_CREAT | O_RDWR | O_WRONLY) > 0
+
   - id: ssl_certificate_tampering
     description: Tampering with SSL certificates for machine-in-the-middle attacks against OpenSSL
     expression: >-
       open.filename =~ "/etc/ssl/certs/*" && open.flags & (O_CREAT | O_RDWR | O_WRONLY) > 0
+
   - id: pci_11_5_critical_binaries
     description: Modification of critical binary files
     expression: >-
-      (open.filename =~ "/bin/*" || 
-      open.filename =~ "/sbin/*" || 
-      open.filename =~ "/usr/bin/*" || 
+      (open.filename =~ "/bin/*" ||
+      open.filename =~ "/sbin/*" ||
+      open.filename =~ "/usr/bin/*" ||
       open.filename =~ "/usr/sbin/*") &&
       open.flags & (O_CREAT | O_RDWR | O_WRONLY) > 0
-  - id: pci_11_5_critical_files
-    description: Modification of critical system files in /opt
-    expression: >-
-      open.filename =~ "/opt/*" &&
-      open.flags & (O_CREAT | O_RDWR | O_WRONLY) > 0 &&
-      process.name not in ["agent", "security-agent", "system-probe", "process-agent", "postgres"]
+      && process.filename not in ["/usr/bin/containerd", "/usr/bin/dpkg"]
```

Comment:
nit: maybe we should add other package manager

Reply:
@safchain happy to! Did you have a specific addition in mind?

Reply:
Might be worth adding a package manager macro, and consider adding yast and yum.